R/tags.R

```diff
@@ -256,6 +256,14 @@ tagList <- function(...) {
   return(lst)
 }
 
+#' @export
+tagFunction <- function(func) {
+  if (!is.function(func) || length(formals(func)) != 0) {
+    stop("`func` must be a function with no arguments")
+  }
+  structure(func, class = "shiny.tag.function")
```

Comment:
```suggestion
  structure(func, class = c("shiny.tag.function", "function"))
```

Reply:
Why? Even without the explicit `"function"` class, `is.function(func)` will still be TRUE.

Reply:
@schloerke recommended it

Reply:
I think we should not have it -- typically if you add a class, you don't also add the base class. I can't think of any cases where a classed list or atomic vector has the base class also explicitly added.